src/OrchardCore.Modules/OrchardCore.Flows/Models/FlowMetadata.cs

```diff
@@ -14,5 +14,6 @@ public class FlowMetadata : ContentPart
     {
         public FlowAlignment Alignment { get; set; } = FlowAlignment.Justify;
         public int Size { get; set; } = 100;
+        public string[] Classes { get; set; }
     }
```

Comment:
You could just add ` = Array.Empty<string>();` here, to avoid null checking it elsewhere (we do this for arrays generally)